go/pkg/api/api.go

```diff
@@ -162,10 +163,14 @@ type TxParam struct {
 // swagger:model TransactionHash
 type TransactionHash string
 
+// swagger:model GasAmount
+type GasAmount string
+
 // BaseAPI interface for all coinstacks to implement
 type BaseAPI interface {
 	GetInfo() (Info, error)
 	GetAccount(pubkey string) (Account, error)
 	GetTxHistory(pubkey string, cursor string, pageSize int) (TxHistory, error)
 	SendTx(hex string) (string, error)
+	EstimateGas(hex string) (string, error)
```

Comment:
I overlooked this throughout the initial reviews. We should not actually add this function to the `BaseAPI` as not all chains will necessarily have this functionality. This will also fix the failing CI since osmosis and thorchain do not implement this currently.
```suggestion
```

Reply:
Yep I remove it.